lib/crudry_query.ex

```diff
@@ -16,22 +16,26 @@ defmodule Crudry.Query do
   * `offset`: defaults to `0`
   * `sorting_order`: defaults to `:asc` (only works if there is also a `order_by` specified)
   * `order_by`: defaults to not ordering
+  * `custom_query`: A functions that receives the initial query and absinthe's info as arguments and returns a custom query. Defaults to initial_query
```

Comment:
We have two different functions, right?

This one receives only the `initial_query` and returns another `query`.

Maybe it would be nice to add Usage examples, here showing a custom query that is independent of absinthe and in the Resolver showing one that uses the context